modules/geo-layers/src/h3-layers/h3-cluster-layer.js

```diff
@@ -1,14 +1,14 @@
 import {h3SetToMultiPolygon} from 'h3-js';
 
-import {CompositeLayer, createIterable} from '@deck.gl/core';
-import {PolygonLayer} from '@deck.gl/layers';
+import {createIterable} from '@deck.gl/core';
+import GeoCellLayer from '../geo-cell-layer/GeoCellLayer';
 
 const defaultProps = {
-  ...PolygonLayer.defaultProps,
+  ...GeoCellLayer.defaultProps,
```

Comment:
Sub class automatically gets all base class props. You do not have to merge them.